Readme.md

```diff
@@ -152,6 +152,16 @@ end
  - make a PR ... get reviewed ... merge
  - datadog is updated by CI
 
+#### Updating existing SLO alerts
```

Comment:
can this just go away since it follows the regular "use import with an url" pattern or is it tricky ?

Reply:
maybe it could go away if we could update the section above with example for monitors and SLO alerts?

```markdown
### Updating an existing monitor
 - use [datadog monitor UI](https://app.datadoghq.com/monitors/manage) to find a monitor
 - get the `id` from the url
 - run `URL='https://app.datadoghq.com/monitors/123' bundle exec rake kennel:import` and copy the output
 ```